code/datums/components/defibrillator.dm

```diff
@@ -304,6 +304,10 @@
 		for(var/obj/item/grab/G in target.grabbed_by)
 			if(ishuman(G.assailant))
 				excess_shock(user, target, G.assailant, defib_ref)
+		if(target.receiving_cpr_from)
+			var/mob/living/L = locateUID(target.receiving_cpr_from)
```

Comment:
Why living type instead of human? receiving_cpr_from will always contain a human right?

Reply:
Yeah, it'll always be a human, I just figured it didn't need to be too restrictive. I'll narrow it down a bit more.